net/cifsd-tools/files/cifsd.init

```diff
@@ -129,20 +131,20 @@ service_triggers()
 start_service()
 {
 	init_config
-	
+
 	if [ ! -e /etc/cifs/smb.conf ]; then
 		logger -t 'cifsd' "missing config /etc/cifs/smb.conf, needs to-be created manually!"
 		exit 1
 	fi
 
 	[ -f /tmp/cifsd.lock ] && rm /tmp/cifsd.lock
-	
+
 	# try remove again before start
-	if (lsmod | grep cifsd &>/dev/null); then
-		rmmod cifsd &>/dev/null
+	if (lsmod | grep cifsd > /dev/null 2>&1); then
```

Comment:
Instead of lsmod, you can use `[ -e /sys/module/cifsd/refcnt ]` to check if it's loaded.

Reply:
I changed it to something similar. Please review.